lib/generators/boring/whenever/install/install_generator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Boring
+  module Whenever
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds whenever to the application"
+
+      def add_whenever_gem
+        say "Adding whenever gem", :green
+
+        Bundler.with_unbundled_env do
+          run "bundle add whenever"
+        end
+
```

Comment:
Let's remove blank line since it's the end of the method after this Bundler block